.github/workflows/ci.yml

```diff
@@ -23,7 +24,6 @@ jobs:
       - name: start mysql
         uses: shogo82148/actions-setup-mysql@6ab6fe9e7aaea1bb6057a1603872c3a6f644f7a3 # v1.38.1
         with:
-          mysql-version: ${{ matrix.mysql }}
```

Comment:
Not sure when this was added but we're not matrix testing mysql versions at this time.